pkg/dispatcher/operator/workers/controller_test.go

```diff
@@ -140,11 +141,14 @@ func TestAssignCalulationDB(t *testing.T) {
 		},
 	}
 
+	var redisSortedSetName = flag.String("redis-sorted-set-name", "vz", "Redis sorted set name")
```

Comment:
No need to add flags in tests. Instead please add the value in the test struct which also gives up the ability to test multiple values.